src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -2,14 +2,13 @@
   
   <PropertyGroup Label="Targets required for unit tests to run">
     <TargetFrameworkNetDesktop461>net461</TargetFrameworkNetDesktop461>
-    <TargetFrameworkNetCore>netcoreapp2.1</TargetFrameworkNetCore>
+    <TargetFrameworkNetCore>netcoreapp3.1</TargetFrameworkNetCore>
```

Comment:
Why was this updated? Were there issues with 2.1?

Reply:
No issues, but 2.1 is out of support.

Reply:
@azchohfi - if MSAL's current targets do not cause an issue with System.Text.Json, I'd like to keep them as they are for now, even if they are deprecated. We will upgrade when needed. Even though they are deprecated, higher level SDKs use them and we'd break them. It's not clear cut if an SDK should target deprecated tfm or not, since apps define the runtime. Better to take it slowly on this aspect.

I guess only net45 needs to go?

Reply:
Sure, no problem. I can roll back the related commits.